src/domain/cohort.js

```diff
@@ -4,28 +4,47 @@ import dbClient from '../utils/dbClient.js'
  * Create a new Cohort in the database
  * @returns {Cohort}
  */
-export async function createCohort() {
+export async function createCohort(name) {
   const createdCohort = await dbClient.cohort.create({
-    data: {}
+    data: { name: `${name}` }
```

Comment:
I don't think we need to be using a template literal here. You can just pass the name variable in